portfolio/src/main/webapp/index.html

```diff
@@ -8,8 +8,8 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Jess's Portfolio</h1>
+      <p>If you're here, you probably already know that I'm a <strong>Google STEP intern</strong>. However, I'm also frequently described as an <strong>Incoming Sophomore at UT Austin</strong>, a <strong>CS and Math Double Major</strong>, a <strong>fraternal twin</strong>, an <strong>amateur unicyclist</strong>, and a <strong>tennis enthusiast</strong>. When I work, I describe myself as a <strong>curious developer</strong>. In the future, I hope to become a <strong>decent ukulelist</strong>, a <strong>strong writer</strong>, and eventually, a <strong>full-time software developer</strong>.
```

Comment:
nit: you may want to line-wrap this at 100 or 120 chars to make reviewing easier (your editor probably auto-wraps long lines, but lots of tools don't)

Reply:
That makes sense. I'll make the change. Thanks!

Reply:
In order to incorporate feedback like this, should I create a new pull request and submit it to you for review again? Or would I go ahead and merge the code, and update the codebase?

Reply:
You'll want to amend the pull request, either by amending one of the existing commits or adding a new commit. Found some discussion here:

https://github.com/github/hub/issues/198

I'm not sure which is preferred -- seems like amending would result in a cleaner history, while adding new commits might make it easier for reviewers to see what has changed since the last round of review. I guess let's go with the latter approach?